llvm/lib/Target/RISCV/RISCVAsmPrinter.cpp

```diff
@@ -981,3 +984,35 @@ bool RISCVAsmPrinter::lowerToMCInst(const MachineInstr *MI, MCInst &OutMI) {
   }
   return false;
 }
+
+static MCSymbolRefExpr::VariantKind
+getModifierVariantKind(RISCVCP::RISCVCPModifier Modifier) {
```

Comment:
Do we need this RISCVCPModifier enum at all? Can we just use `MCSymbolRefExpr::VK_None` always?